Makefile

```diff
@@ -1,23 +1,72 @@
-local:
+# Set up environment
+environment:
+	npm ci
+
+# Build Product or Community
+local-product:
+	mkdir -p tmp
+	bin/switch-prod-comm product
+	npx antora --version
+	npx antora --stacktrace --log-format=pretty --log-level=info \
+		turtles-local-product-playbook.yml \
+		2>&1 | tee tmp/local-product-build.log 2>&1
+
+local-community:
 	mkdir -p tmp
+	bin/switch-prod-comm community
 	npx antora --version
 	npx antora --stacktrace --log-format=pretty --log-level=info \
-		turtles-local-playbook.yml \
-		2>&1 | tee tmp/local-build.log 2>&1
+		turtles-local-community-playbook.yml \
+		2>&1 | tee tmp/local-community-build.log 2>&1
+
+remote-product:
+	mkdir -p tmp
+	npm ci
```

Comment:
`remote-product` and `remote-community` are the only two with `npm ci` included, the other four targets do not include it. 

The process should be consistent across the board to avoid confusion/inconsistencies. They should either all handle it for the users or all assume the user handles it manually. If the former, you should list it as a prerequisite for each target since `environment` is already defined.

e.g.

`local-product: environment`